python/pipeline/pupil.py

```diff
@@ -540,4 +575,998 @@ def make(self, key):
 
         self.insert1(key)
         for ckey in tqdm(contours):
-            self.Ellipse().insert1(ckey, ignore_extra_fields=True)
\ No newline at end of file
+            self.Ellipse().insert1(ckey, ignore_extra_fields=True)
+
+# If config.yaml ever updated, make sure you store the file name differently so that it becomes unique
+@schema
+class ConfigDeeplabcut(dj.Manual):
+    definition = """
+    # Minimal info needed to load deeplabcut model
+    config_path         : varchar(255)          # path to deeplabcut config
+    ---
+    shuffle             : smallint unsigned     # shuffle number used for the trained dlc model. Needed for dlc.analyze_videos
+    trainingsetindex    : smallint unsigned     # trainingset index used for the trained dlc. model. Needed for dlc.analyze_videos
+    """
+
+@schema
+class TrackingMethod(dj.Manual):
+    definition="""
+    -> Eye
+    tracking_method     : tinyint unsigned # tracking_method 0:manual 1: deeplabcut
+    ---
+    """
+
+    def fill(self, key, tracking_method):
+        self.insert1(dict(key, tracking_method=tracking_method))
+
+@schema
+class Tracking(dj.Computed):
+    definition="""
```

Comment:
Eye should be here:
```python
class Tracking(dj.Computed)
 -> Eye
->TrackingMethod
---
tracking_ts
```
And then people can call it as Trakcing.populate({'animal_id': slslls, 'session': slslls, 'scan_idx': sksks, 'tracking_method': slslsl}). And when we add it to the populate script, it will just be `pupil.Tracking(next_scans, {'tracking_method': 1}) #assuming 1 is deeplabcut`